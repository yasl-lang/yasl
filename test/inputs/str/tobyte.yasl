
echo .A->tobyte()
echo .a->tobyte()
echo ' '->tobyte()
echo '~'->tobyte()
