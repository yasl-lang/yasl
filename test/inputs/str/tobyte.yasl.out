65
97
32
126
